components/global.css

```diff
@@ -35,6 +26,9 @@ p {
 .ml-1 {
 	margin-left: 1rem;
 }
+.ml-2 {
```

Comment:
Not that it needs to be done in this PR, but it might be cool to do something like this at some point: https://epsi-rns.gitlab.io/frontend/2019/06/21/sass-loop-spacing-class/

Reply:
I like that idea too.